chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -181,7 +183,12 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
         });
     }, [props.webChatContainerProps?.webChatStyles]);
 
-
+    useEffect(() => {
```

Comment:
This can be combined with another already existed useEffect?

Reply:
done